### PR TITLE
fix(bootstrap): only bootstrap the first time

### PR DESCRIPTION
**Pull request by WikiRik**
@linuxbandit this good? Or should we do something else in the Makefile depending on the MYAEGEE_ENV in the .env file?

### CODE REVIEW

**Comment by WikiRik** [comment]
Okay, I've updated the MYAEGEE_ENV because I think for the longer run it is better to build all containers locally immediately in case of any errors. Will take more time on the first thing, but not on the longer run

File: vagrant-post-script/orchestrate_docker.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ chmod +x helper.sh
 find . -type f -name "*.sh" -exec dos2unix {} \+;
 dos2unix .env Makefile
 
-if [[ -f .env ]]; then
+if [[ -f .init ]]; then
```
Comment (linuxbandit) [inline vagrant-post-script/orchestrate_docker.sh:14]: the file .init is never created anywhere

**Comment by linuxbandit** [comment]
> Okay, I've updated the MYAEGEE_ENV because I think for the longer run it is better to build all containers locally immediately in case of any errors. Will take more time on the first thing, but not on the longer run

hmmm i am on the fence for this

File: vagrant-post-script/orchestrate_docker.sh (hunk 1 above)
Comment (WikiRik) [reply vagrant-post-script/orchestrate_docker.sh:14]: https://github.com/AEGEE/MyAEGEE/blob/master/helper.sh#L74 as part of make init

**Comment by WikiRik** [comment]
I understand, and maybe we can find out an intermediate solution soon. But currently the initial setup takes too long with the core seeding

**Comment by WikiRik** [comment]
@linuxbandit I would prefer to merge this before the hackathon, are you okay with me merging this? We'll find out a better solution later

**Review by linuxbandit — APPROVED** [review]

**Pull request merged**